gnd/src/main/res/values-pt/strings.xml

```diff
@@ -174,43 +174,4 @@ Tem certeza que deseja apagar o local e todos os dados associados? Isso não pod
   <!-- Precedes the build number shown in the side menu of debug builds. -->
   <string name="build">Construir %s</string>
 
-  <string name="continue_editing">Continuar editando</string>
```

Comment:
Looks like you deleted all the Portuguese translations by accident.. Please add them back.

Reply:
that why do to rebase got repeated so deleted only repeated ones.